src/bun.js/api/server.zig

```diff
@@ -5119,7 +5127,7 @@ pub const ServerWebSocket = struct {
         return ZigString.init(text).toValueGC(globalThis);
     }
 };
-
+extern fn RequestPrototype__signalGetCachedValue(JSC.JSValue) JSC.JSValue;
```

Comment:
Can you add a method to `Request` instead of duplicating this method in `server.zig`?